src/main/java/com/neroreflex/plugins/PTforum.java

```diff
@@ -31,18 +31,21 @@
  */
 public final class PTforum extends Trancio {
 
-    String apiEndpoint = "http://www.pierotofy.it/pages/extras/forum/api/last_topics.php";
+    static String apiEndpoint = "http://www.pierotofy.it/pages/extras/forum/api/last_topics.php";
 
     HashMap<String, Integer> topics = new HashMap<>(); //chiave=url, valore=numero risposte
 
     @Override
     public final void onPoll(){
         try {
+            Vector<String> joinedChannels = this.getChannels();
+
             // Ottengo la lista di canali in cui informare gli utenti
-            String[] channels = this.getChannels();
+            String[] channels = new String[joinedChannels.size()];
+            joinedChannels.copyInto(channels);
```

Comment:
Il foreach si può usare su tutte le collection iterabili mi pare, quindi non serve che copi joinedChannels in un array (mi era sfuggito nella prima review).